x-pack/plugins/ml/server/plugin.ts

```diff
@@ -220,15 +227,15 @@ export class MlServerPlugin
         coreSetup.getStartServices
       ),
       mlLicense: this.mlLicense,
-      getEnabledFeatures: () => Object.assign({}, this.enabledFeatures),
+      getEnabledFeatures: () => Object.assign({}, this.enabledFeatures), // object.freeze()?
```

Comment:
Commented unused code

Reply:
Is there a reason we are creating a new object here for `getEnabledFeatures` instead of returning the already frozen object??

Reply:
Good spot, no this isn't need. I meant to remove it, hence the commented code.

Reply:
Updated in https://github.com/elastic/kibana/pull/168721/commits/781a421d714b43d1ed607287184590a5a4b10350